docker-compose.yaml

```diff
@@ -36,28 +36,31 @@ services:
       DATABASE_URL: postgresql+asyncpg://postgres:postgres@db:5432/postgres
       PORT: "80"
 
-  fake_govuk:
-    container_name: fake_govuk
+  dummy-service-1:
+    container_name: dummy-service-1
     networks:
       - e2e
-    image: "gcr.io/sde-consent-api/fake_govuk:latest"
-    ports:
-      - "8001:80"
+    image: "${DOCKER_IMAGE}"
+    volumes:
+      - "./client:/home/app/client"
```

Comment:
Yeah, that might be helpful 👍

Reply:
`consent_api/` is already in /home/app, but mounting the current codebase over the top might be useful for making changes while docker image is running. It would be good to have a way to get gunicorn to reload the code too, maybe SIGHUP or SIGUSR2 might work for this. https://docs.gunicorn.org/en/stable/signals.html